appion/wizards.py

```diff
@@ -35,7 +35,8 @@
 from pyworkflow.em import CoordinatesObjectView
 from pyworkflow.utils import makePath, cleanPath, readProperties
 
-from protocol_dogpicker import DogPickerProtPicking
+from protocols.protocol_dogpicker import DogPickerProtPicking
```

Comment:
I think this import could also be changed to:
from appion.protocols import DogPickerProtPicking
The rationale is:
* You don't assume protocols location relative to wizards module
* You don't assume the filename implementation is protocol_dogpicker inside the protocols submodule.
* So, the only granted thing here (and for protocols inside a plugin) is that they should be located inside the package.protocols submodule (appion.protocols in this case).